test/core/align.wast

```diff
@@ -856,5 +856,11 @@
   (func (export "store") (param i32 i64)
     (i64.store align=4 (get_local 0) (get_local 1))
   )
+  (func (export "load") (param i32) (result i32)
+    (i32.load (get_local 0))
+  )
 )
-(assert_trap (invoke "store" (i32.const 65531) (i64.const 1)) "out of bounds memory access")
+
+(assert_trap (invoke "store" (i32.const 65532) (i64.const 1)) "out of bounds memory access")
```

Comment:
Perhaps better store -1, so that all bits would be set.

Reply:
Indeed, I just updated it. :)